src/Stripe.Tests/transfers/when_creating_a_transfer_with_application_fee.cs

```diff
@@ -0,0 +1,29 @@
+﻿// ***** This test needs a bank account setup in order to run correctly, and the ignore attribute wasn't working. Good times.
```

Comment:
Sorry that was copypasta from the other test. Realized I didn't rename the class. Removed behaviours. Not sure I can remove the protected types.